File: invesdwin-context-parent/invesdwin-context-javascript/src/test/java/de/invesdwin/context/javascript/tests/InputsAndResultsTestFloat.js
print("getFloat")
if(typeof getFloat !== 'undefined')
	throw "getFloat already defined!"
getFloat = putFloat
getFloatType = typeof getFloat
print(getFloatType)
print(getFloat)
if(getFloatType !== 'number')
	throw "getFloat not number!"

print("getFloatVector")
if(typeof getFloatVector !== 'undefined')
	throw "getFloatVector already defined!"
getFloatVector = putFloatVector
getFloatVectorType = typeof getFloatVector[0]
print(getFloatVectorType)
print(getFloatVector)
if(getFloatVectorType !== 'number')
	throw "getFloatVector not number!"

print("getFloatVectorAsList")
if(typeof getFloatVectorAsList !== 'undefined')
	throw "getFloatVectorAsList already defined!"
getFloatVectorAsList = putFloatVectorAsList
getFloatVectorAsListType = typeof getFloatVectorAsList[0]
print(getFloatVectorAsListType)
print(getFloatVectorAsList)
if(getFloatVectorAsListType !== 'number')
	throw "getFloatVectorAsList not number!"

print("getFloatMatrix")
if(typeof getFloatMatrix !== 'undefined')
	throw "getFloatMatrix already defined!"
getFloatMatrix = putFloatMatrix
getFloatMatrixType = typeof getFloatMatrix[0][0]
print(getFloatMatrixType)
print(getFloatMatrix)
if(getFloatMatrixType !== 'number')
	throw "getFloatMatrix not number!"

print("getFloatMatrixAsList")
if(typeof getFloatMatrixAsList !== 'undefined')
	throw "getFloatMatrixAsList already defined!"
getFloatMatrixAsList = putFloatMatrixAsList
getFloatMatrixAsListType = typeof getFloatMatrixAsList[0][0]
print(getFloatMatrixAsListType)
print(getFloatMatrixAsList)
if(getFloatMatrixAsListType !== 'number')
	throw "getFloatMatrixAsList not number!"

File: invesdwin-context-parent/invesdwin-context-jruby/src/test/java/de/invesdwin/context/jruby/tests/InputsAndResultsTestFloat.rb
puts("getFloat")
unless (defined?(getFloat)).nil? || getFloat.nil?
	raise "getFloat already defined!"
end
getFloat = putFloat
getFloatType = getFloat.class
puts(getFloatType)
puts(getFloat)
if(getFloatType != Float)
	raise "getFloat not Float!"
end

puts("getFloatVector")
unless (defined?(getFloatVector)).nil? || getFloatVector.nil?
	raise "getFloatVector already defined!"
end
getFloatVector = putFloatVector
getFloatVectorType = getFloatVector[0].class
puts(getFloatVectorType)
puts(getFloatVector)
if(getFloatVectorType != Float)
	raise "getFloatVector not Float!"
end

puts("getFloatVectorAsList")
unless (defined?(getFloatVectorAsList)).nil? || getFloatVectorAsList.nil?
	raise "getFloatVectorAsList already defined!"
end
getFloatVectorAsList = putFloatVectorAsList
getFloatVectorAsListType = getFloatVectorAsList[0].class
puts(getFloatVectorAsListType)
puts(getFloatVectorAsList)
if(getFloatVectorAsListType != Float)
	raise "getFloatVectorAsList not Float!"
end

puts("getFloatMatrix")
unless (defined?(getFloatMatrix)).nil? || getFloatMatrix.nil?
	raise "getFloatMatrix already defined!"
end
getFloatMatrix = putFloatMatrix
getFloatMatrixType = getFloatMatrix[0][0].class
puts(getFloatMatrixType)
puts(getFloatMatrix)
if(getFloatMatrixType != Float)
	raise "getFloatMatrix not Float!"
end

puts("getFloatMatrixAsList")
unless (defined?(getFloatMatrixAsList)).nil? || getFloatMatrixAsList.nil?
	raise "getFloatMatrixAsList already defined!"
end
getFloatMatrixAsList = putFloatMatrixAsList
getFloatMatrixAsListType = getFloatMatrixAsList[0][0].class
puts(getFloatMatrixAsListType)
puts(getFloatMatrixAsList)
if(getFloatMatrixAsListType != Float)
	raise "getFloatMatrixAsList not Float!"
end
